nsv13/code/modules/overmap/types/dominion.dm

```diff
@@ -0,0 +1,43 @@
+/obj/structure/overmap/nanotrasen/serendipity
```

Comment:
As this is a nanotrasen subtype this needs to be in the nanotrasen file.

Reply:
Done and committed. Likewise moved the .dmi file into the nanotrasen folder.